internal/website/src/components/MetaTags.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
You will probably want to do a git merge git push here to remove those merged changes

Reply:
Oops! Fixed.